code/modules/nanogate/nanogate_items.dm

```diff
@@ -5,18 +5,19 @@
 	icon_state = "ihs_rig_old"
 	suit_type = "nanite suit"
 	armor_list = list(
-		melee = 30,
-		bullet = 30,
-		energy = 30,
+		melee = 35,
+		bullet = 35,
+		energy = 35,
 		bomb = 25,
 		bio = 100,
 		rad = 100
 	)
-	slowdown = 0.3
+	slowdown = 0.15 //Less slowdown cause it's light
 	item_flags = STOPPRESSUREDAMAGE | THICKMATERIAL | DRAG_AND_DROP_UNEQUIP | EQUIP_SOUNDS
 	offline_slowdown = 2
 	offline_vision_restriction = 0
 	drain = 2
+	max_upgrades = 2 //Let's add an extra upgrade slot since you're stuck with this thing
```

Comment:
please directly test this bit
as their can be issues with adding mods to space suits/rigs and still intracting with them with tools/fixing/ect

Reply:
Had to be removed due to an exploit I found that can be done with rigs that have 2 upgrade slots.

Reply:
yeah having extra upgrades on a RIG breaks things 100%. I tried it before